Dockerfile

```diff
@@ -1,12 +1,5 @@
-FROM jimmycuadra/rust:1.16.0
+FROM debian:jessie-slim
```

Comment:
Yep, good points. I looked into using alpine, but I couldn't get our project to build against the musl implementation of glibc. Specifically, we depend on rustls which depends on ring, and ring fails to build in all of the alpine-based images that I tried. ring does build successfully in the `jimmycuadra/rust` image, which uses jessie as a base image. So swapping in jessie-slim here works and everything links correctly.